### PR TITLE
fix(node): TLSSocket._start

### PR DESCRIPTION
**Pull request by bartlomieju**
Closes https://github.com/denoland/deno/issues/19983
Closes https://github.com/denoland/deno/issues/18303
Closes https://github.com/denoland/deno/issues/16681
Closes https://github.com/denoland/deno/issues/19978

### CODE REVIEW

File: ext/node/polyfills/_tls_wrap.ts [hunk 1]
```diff
@@ -19,6 +19,7 @@ import {
   constants as TCPConstants,
   TCP,
 } from "ext:deno_node/internal_binding/tcp_wrap.ts";
+import { kHandle } from "ext:deno_node/internal/stream_base_commons.ts";
```
Comment (ije) [inline ext/node/polyfills/_tls_wrap.ts:22]: @bartlomieju can you please also fix it as 0.177.2 for esm.sh? thanks!

Comment (ije) [reply ext/node/polyfills/_tls_wrap.ts:22]: makes sense

**Comment by bartlomieju** [comment]
Reproduction
```js
import nodemailer  from "nodemailer";

nodemailer.createTestAccount((err, account) => {
    if (err) {
        console.error('Failed to create a testing account. ' + err.message);
        return process.exit(1);
    }

    console.log('Credentials obtained, sending message...');

    // Create a SMTP transporter object
    let transporter = nodemailer.createTransport({
        host: account.smtp.host,
        port: account.smtp.port,
        secure: account.smtp.secure,
        auth: {
            user: account.user,
            pass: account.pass
        }
    });

    console.log("transporter", transporter);

    // Message object
    let message = {
        from: 'Sender Name <sender@example.com>',
        to: 'Recipient <recipient@example.com>',
        subject: 'Nodemailer is unicode friendly ✔',
        text: 'Hello to myself!',
        html: '<p><b>Hello</b> to myself!</p>'
    };
    console.log("before send");
    transporter.sendMail(message, (err, info) => {
        console.log("after send");
        if (err) {
            console.log('Error occurred. ' + err.message);
            return process.exit(1);
        }

        console.log('Message sent: %s', info.messageId);
        // Preview only available when sending through an Ethereal account
        console.log('Preview URL: %s', nodemailer.getTestMessageUrl(info));
    });
});
```

**Comment by bartlomieju** [comment]
Closing in favor of https://github.com/denoland/deno/pull/20120

**Pull request closed without merging**